changelog/_unreleased/2020-09-17-add-autoscroll-to-listing.md

```diff
@@ -0,0 +1,9 @@
+---
+title:              Add autoscroll to listing
+issue:              NEXT-9021
+author:             Sebastian König
+author_email:       s.koenig@tinect.de
+author_github:      @tinect
+---
+# Storefront
+*  Changed `listing.plugin.js` to scroll to top of listing when it is updating
```

Comment:
-updating +updated

Reply:
No, it scrolls immediately :-) Not when the content is there.